pr_agent/tools/pr_reviewer.py

```diff
@@ -100,9 +101,33 @@ async def run(self) -> None:
             if self.is_auto and not get_settings().pr_reviewer.automatic_review:
                 get_logger().info(f'Automatic review is disabled {self.pr_url}')
                 return None
-            if self.is_auto and self.incremental.is_incremental and not self.incremental.first_new_commit_sha:
-                get_logger().info(f"Incremental review is enabled for {self.pr_url} but there are no new commits")
-                return None
+            if self.incremental.is_incremental:
```

Comment:
@zmeir  please move all of this to a dedicated, self-contained function.

this is too much content to appear directly